docs/api.md

```diff
@@ -679,6 +693,24 @@ Destroy the editor. Clean up DOM, event listeners, and web workers.
 
 Expand all fields. Only applicable for mode 'tree', 'view', and 'form'.
 
+#### `JSONEditor.expand(path, isExpand, recursive)`
```

Comment:
Thinking aloud here: it may be nice to change the signature 

```js
JSONEditor.expand(path, isExpand, recursive) 
```
to
```js
JSONEditor.expand({path, isExpand, recursive}) 
```

Then you can pass the options even more easily from one editor to an other:

```js
const editor1 = new JSONEditor(container1, { onExpand: (props) => editor2.expand(props) })
const editor2 = new JSONEditor(container2, { onExpand: (props) => editor1.expand(props) })
```

What do you think?